frontend/src/pages/Favourite/Favourite.css

```diff
@@ -0,0 +1,7 @@
+.title{
+    font-weight: bold;
+}
+
+.FavouritesList{
+    margin-bottom: 10%;
```

Comment:
% mesurements in css are a bad idea most of the time. increases the fideling. Just saying.